CHANGELOG.md

```diff
@@ -6,6 +6,24 @@
 
 ### Fixes
 
+## Next v1.28.0 (2022-10-31)
+
+### Fixes
+
+- babel : migrate to the top level setPublicClassFields assumption ([@mmorel-35](https://github.com/mmorel-35) in [#1008](https://github.com/jaegertracing/jaeger-ui/pull/1008))
+
+- Add documentation to config ([@yurishkuro](https://github.com/yurishkuro) in [#1007](https://github.com/jaegertracing/jaeger-ui/pull/1007))
+
+- Bump async from 2.6.2 to 2.6.4 ([@dependabot](https://github.com/dependabot) in [#1002](https://github.com/jaegertracing/jaeger-ui/pull/1002))
```

Comment:
Good point, the previous releases except 1.27.1 didn't list dependencies bumps. However, the rule is not documented anywhere. 

Since there is less feature work, I would propose listing dependencies bumps (except updating GHA versions).

Reply:
My vote is to exclude version bumps (unless if they relate to specific issues) because they make it hard to spot features/fixes contributed by the community.

I think it made sense to include the dependency bump in 1.27.1 because (my guess is) the new versions fix a specific known issue that directly impacted Jaeger UI.

If there are no features/fixes contributed during the month, I think it's okay to use a placeholder to indicate no relevant Jaeger UI changes were made.

That's just my two-cents worth; not sure what others (esp. @yurishkuro) think.

> the rule is not documented anywhere.

I just applied the same changelog grooming practices from jaegertracing/jaeger to this repo. Good point though, I think either way, maybe we should more explicitly document the what should and should not be included.